cmd/mobynit/main.go

```diff
@@ -56,6 +49,22 @@ func mountContainer(containerID, graphDriver string) string {
 	if err := unix.Mount("", newRootPath, "", unix.MS_REMOUNT, ""); err != nil {
 		log.Fatal("error remounting container as read/write:", err)
 	}
+
+	return newRootPath
+}
+
+func prepareForPivot(sysroot, containerID, graphDriver string) string {
```

Comment:
This is only ever called with `sysroot` set to the empty string. What's the reason for the parameter?

Reply:
Removed the parameter